gsitemap.php

```diff
@@ -623,7 +639,7 @@ protected function _getSupplierLink(&$link_sitemap, $lang, &$index, &$i, $id_sup
             if ($file_headers[0] != 'HTTP/1.1 404 Not Found' || $file_headers === true) {
                 $supplier_image = array(
                     'title_img' => htmlspecialchars(strip_tags($supplier->name)),
-                    'link' => 'http'.(Configuration::get('PS_SSL_ENABLED') ? 's' : '').'://'.Tools::getMediaServer(_THEME_SUP_DIR_)._THEME_SUP_DIR_.((!file_exists(_THEME_SUP_DIR_.'/'.(int)$supplier->id.'-'.ImageType::getFormattedName('medium').'.jpg')) ? $lang['iso_code'].'-default' : (int)$supplier->id).'-'.ImageType::getFormattedName('medium').'.jpg'
+                    'link' => $this->context->link->getBaseLink().Tools::getMediaServer(_THEME_SUP_DIR_)._THEME_SUP_DIR_.((!file_exists(_THEME_SUP_DIR_.'/'.(int)$supplier->id.'-'.ImageType::getFormattedName('medium'))) ? $lang['iso_code'].'-default' : (int)$supplier->id).'-'.ImageType::getFormattedName('medium')
```

Comment:
Same comment about `$this->context->link->getBaseLink()`

Reply:
Please check this commit. I removed supplier and manufacturer because they are not used as pages I think?

Reply:
https://github.com/PrestaShop/gsitemap/pull/53/commits/91de6e87abad28c80692137bab532cf1e392ee13